espq.py

```diff
@@ -141,8 +148,11 @@ def __init__(self, device):
             bf_str = ('-DWIFI_SSID=\\"{wifi_ssid}\\" '
                       '-DWIFI_PASSWORD=\\"{wifi_pass}\\" '
                       '-DNAME=\\"{name}\\" '
-                      '-DMQTT_SERVER=\\"{mqtt_host}\\" '
-                      '-DTOPIC=\\"{base_topic}/{topic}\\"')
+                      '-DMQTT_SERVER=\\"{mqtt_host}\\" ')
+            if self._key_exists('base_topic'):
+                bf_str += '-DTOPIC=\\"{base_topic}/{topic}\\"'
```

Comment:
Shouldn't this just be topic either way, because up at line 109 topic is defined to include base_topic?

Reply:
Oh, you're right. I thought too hard about it and double corrected...